client/src/components/loginform.js

```diff
@@ -7,14 +7,14 @@ const LoginForm = ()=> {
         <div className="loginform">
             <img src={loginavi} alt="login avi" />
             <h1>Inforum</h1>
-            <h3>Give your ideas a voice.</h3>
+            <h2>Give your ideas a voice.</h2>
             <input type="text"placeholder="Username" />
             <input type="password"placeholder="Password" />
 
             <div>
             <button type="button" className="login-btn">Continue</button>
```

Comment:
In the form you will create. Replace this with a <input type ="submit value="Logi In">

Reply:
done